src/timestamp.cpp

```diff
@@ -0,0 +1,53 @@
+#include "g3log/time.hpp"
+#include "g3log/timestamp.hpp"
+#include <sstream>
+
+
+namespace g3 {
+
+
+	Timestamp::Timestamp(std::string& defaultFormat) :
```

Comment:
Update: I made quick and dirty comparison of timestamp fractional part generated by clock_gettime(CLOCK_REALTIME, &_timestamp) with timestamps generated by chrono library:
std::chrono::high_resolution_clock::now().time_since_epoch().count();
And I got consistant results :)

So I will just refine timestamp generation code and enhance formating functionality to support fractional part.